docs/staticAnalysis.md

```diff
@@ -102,6 +118,12 @@ To set it up, [install node.js](https://nodejs.org/en/download/) if necessary (v
 # Remember to use the correct tool version
 npm install -g eslint@{version}
 ```
+The plugin for IntelliJ can be found  [here](https://plugins.jetbrains.com/idea/plugin/7494-eslint).
+
+#### Configuring ESLint for IntelliJ
+
+1. Refer to [this guide](https://github.com/idok/eslint-plugin/blob/master/README.md#settings) to set up the ESLint plugin.
+1. Set the `Rules directory` to the location of `teammates-eslint.yml`.
```

Comment:
Hmm have you installed node.js and eslint? I realised the current version of IntelliJ has incorporated the ESLint plugin. Will update the guide to set that up instead.